functions/go/Makefile

```diff
@@ -26,6 +26,7 @@ FUNCTIONS := \
 	format \
 	gatekeeper \
 	remove-local-config-resources \
+	generate-services \	
```

Comment:
> `	generate-services \	`

There is an empty `\t` after `\`. It should be removed. It is causing the CI to fail.

Also please sort the list.